examples/cms-ghost/README.md

```diff
@@ -0,0 +1,90 @@
+# A statically generated blog example using Next.js and Ghost
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Ghost](https://ghost.org/) as the data source.
+
+> This boilerplate demonstrates simple usage and best practices. If you are looking for a more feature richt Next.js generator for Ghost including the Casper theme,
+> check out [next-cms-ghost](https://github.com/styxlab/next-cms-ghost).
+
+## Demo
+
+[https://ghost-next-blog.vercel.app/](https://ghost-next-blog.vercel.app/)
```

Comment:
This 404s for me.

Reply:
I did not set this up, because I want you to give the option to host this yourself. Let me know.